Hub/Dockerfile

```diff
@@ -16,7 +16,7 @@ ENV GRID_NEW_SESSION_WAIT_TIMEOUT -1
 ENV GRID_JETTY_MAX_THREADS -1
 ENV GRID_NODE_POLLING  5000
 ENV GRID_CLEAN_UP_CYCLE 5000
-ENV GRID_TIMEOUT 30000
+ENV GRID_TIMEOUT 30
```

Comment:
that's ok :)  that's how they're meant to be used.  since this is an environment variable passed to the container, you can set it to whatever you need it to be.  it would be hard for this default value to match "everybody's" standards.